client/layout/masterbar/oauth-client.scss

```diff
@@ -202,11 +203,12 @@
 	}
 
 	.login__form-action,
-	.login__social-buttons {
+	.login__social-buttons,
+	.magic-login__form-action {
 		text-align: center;
 	}
 
-	.wp-login__main.main {
+	.wp-login__main.main, .magic-login.main {
```

Comment:
Obviously super minor, but according to our [coding conventions](https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/#structure) this selector should be on a new line.